packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -13,13 +13,18 @@ URL: https://github.com/awslabs/soci-snapshotter
 Source0: https://github.com/awslabs/soci-snapshotter/archive/v%{gover}/soci-snapshotter-%{gover}.tar.gz
 Source1: bundled-soci-snapshotter-%{gover}.tar.gz
 Source2: bundled-cmd.tar.gz
+Source3: soci-config-toml
+# Mount for writing soci configuration
+# Generated with systemd-escape -p --suffix=mount "/etc/soci-snapshotter"
+Source100: "etc-soci\\x2dsnapshotter.mount"
```

Comment:
What we've done elsewhere - `release.spec` has lots of examples, and `host-ctr.spec` - is to name the mount unit something more natural, like `etc-soci-snapshotter.mount.in`, and then rename it via `systemd-escape` in the spec file.

Reply:
I'll declare a macro like

```
%global soci_mount_path %(systemd-escape --path %{_sysconfdir}/soci-snapshotter)
```

to use in `%install` and `%files`